com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -121,11 +128,10 @@ private bool ComputeRequestIsFree(Vector3 capturePosition)
         /// <returns>Whether the request for light probe rendering has been fulfilled and sh is valid.</returns>
         public bool RetrieveProbeSH(int requestID, out SphericalHarmonicsL2 sh)
         {
-            if (requestID >= 0 && requestID < m_SHCoefficients.Count
-                && ComputeCapturePositionIsValid(m_RequestPositions[requestID]))
+            if (requestID >= 0 && requestID < m_SHCoefficients.Count && ComputeCapturePositionIsValid(m_RequestPositions[requestID]))
             {
                 sh = m_SHCoefficients[requestID];
-                return true;
+                return m_SHValidity[requestID] == kValidSH;
```

Comment:
Not 100% certain if we should require full validity or also allow a threshold of error here.

Reply:
It really depends on the values returned by the lightmapper on non-completely invalid probe locations; I'd say for now we can force it to be required to be fully valid.